lua/orgmode/init.lua

```diff
@@ -20,6 +20,7 @@ local auto_instance_keys = {
 ---@field capture OrgCapture
 ---@field clock OrgClock
 ---@field completion OrgCompletion
+---@field links OrgLinkHandlerRegistry
```

Comment:
Let's name this just `OrgLinks`
```suggestion
---@field links OrgLinks
```